scene/gui/tree.h

```diff
@@ -45,11 +45,11 @@ class TreeItem : public Object {
 
 public:
 	enum TreeCellMode {
-		CELL_MODE_STRING, ///< just a string
-		CELL_MODE_CHECK, ///< string + check
-		CELL_MODE_RANGE, ///< Contains a range
-		CELL_MODE_ICON, ///< Contains an icon, not editable
-		CELL_MODE_CUSTOM, ///< Contains a custom value, show a string, and an edit button
+		CELL_MODE_STRING, // Just a string.
+		CELL_MODE_CHECK, // String and check.
+		CELL_MODE_RANGE, // Contains a range.
```

Comment:
```suggestion
		CELL_MODE_RANGE, // Contains a slider or combo box.
```
Although these comments could be removed; the values are documented and the descriptions are not useful here.